src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -288,12 +302,12 @@ export default function ApplicationForm({
         organizationFacility
       });
 
-      history.push("/");
+      // TODO: Redirect to bambora here...
     }
   };
 
   const back = () => {
-    history.push("/");
+    setToHome(true);
```

Comment:
why not just do the redirect here? it is the only place this state seems to be set and it is only used once so should be able to simplify it by putting the redirect inside here

Reply:
I tried putting the redirects in those functions in the same form as they are in the ifs but they did nothing. I think because there was no state change, there was no rerender, so while I was returning the <Redirect /> it wasn't doing anything.